src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -130,11 +132,11 @@ public Optional<Person> findPersonToEdit(List<Person> persons) throws CommandExc
      * Creates and returns a {@code Person} with the details of {@code personToEdit}
      * edited with {@code editPersonDescriptor}.
      */
-    private static Person createEditedPerson(Person personToEdit, EditPersonDescriptor editPersonDescriptor) {
+    static Person createEditedPerson(Person personToEdit, EditPersonDescriptor editPersonDescriptor) {
```

Comment:
Was there a particular reason to change the access level of the method from `private` to `package private`? From what I see, this method is exclusively used internally by the `EditCommand` class.

Reply:
My bad, I meant to change it to a public method for testing.